spec/rubocop/cop/rspec/verified_doubles_spec.rb

```diff
@@ -53,14 +53,27 @@
     end
   end
 
-  it 'ignores doubles without a name' do
-    expect_no_offenses(<<-RUBY)
+  it 'finds a double when the no name is specified' do
```

Comment:
this one needs improving. Perpahs `finds a double without a name` is enough

Reply:
`doubles that have no name specified` also works, as it is consistent with the other test

Reply:
👍 fixed that up.